webdriver/tests/add_cookie/add.py

```diff
@@ -78,7 +82,8 @@ def test_add_cookie_for_ip(session, url, server_config, configuration):
         "domain": "127.0.0.1",
         "path": "/",
         "httpOnly": False,
-        "secure": False
+        "secure": False,
+        "sameSite": "Lax"
```

Comment:
@christian-bromann could you please add a separate parameterized test for all the available values of same site please? If you don't have the time please let us know and we can cover the test additions with https://bugzilla.mozilla.org/show_bug.cgi?id=1606439.

Also we don't check for invalid values yet, and getCookies also needs a test.